src/resources/Sources/Sources.ts

```diff
@@ -0,0 +1,10 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+
+export default class Sources extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/sources`;
+
+    getItemTypes(sourceId: string) {

```

Comment:
In other resources we use the term `list` instead of `get`. Also, this call is made in the scope of a single source. So would that makes sense to rename the method for `listSourceItemTypes` ?

Reply:
Yes, you're totally right.